packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -28,27 +27,55 @@ while getopts "r:" option; do
     esac
 done
 
-if [ -z "$resourceGroupName" ]; then
+if [ -z $resourceGroupName ]; then
     exitWithUsageInfo
 fi
 
+# Will return the function name if we deployed before the function app template on this resource group
+functionAppName=$(az group deployment show -g "$resourceGroupName" -n "function-app-template"  --query "properties.parameters.name.value" -o tsv)
+
+# Boolean to track if we should create a new App Registration or there's one already existing
+createNewApp=true
```

Comment:
this is just a string "true" right?

Reply:
Yeah, but that's a way to handle Boolean in bash